.github/code/nipoppy_track_processing_for_dataset.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -euo pipefail
+
+DS_ID=$1
+DS_URL="https://github.com/OpenNeuroDatasets/${DS_ID}.git"
+
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
```

Comment:
:cherries: if you're already going through the trouble of setting such a base working dir, why not make the other paths relative to it?

Reply:
The other paths end up being longer as relative paths, since they're all in the repo root where as the scripts are inside `.github/code`! I tried to go with the approach that would result in having to use the least number of `../..` for readability.